raddb/mods-config/redis/ippool/preamble.lua

```diff
@@ -0,0 +1,229 @@
+-- must match redis_ippool.h
+local ippool_rcode_success = 0
```

Comment:
maybe:

```
local ippol_rcode = {
   not_found = -1,
   device_mismatch = -3
   ..........
}
```

then can call using: `ippol_rcode.notfound`

Reply:
As all the logic is in a single small file I do not think there is no gain to readability and indirect table access requires an additional load (not checked the JIT output but guessing it turns directs variables into immediates) via the call to `GETTABLE`.

I think this boils down to coding style, so happy to change it.

```
$ cat example-direct.lua 
local rcode_one = 1
local rcode_two = 2
local rcode_three = 3

print(rcode_one)

$ luac -l example-direct.lua 
main <example-direct.lua:0,0> (7 instructions, 28 bytes at 0x558023218550)
0+ params, 5 slots, 0 upvalues, 3 locals, 4 constants, 0 functions
        1       [1]     LOADK           0 -1    ; 1
        2       [2]     LOADK           1 -2    ; 2
        3       [3]     LOADK           2 -3    ; 3
        4       [5]     GETGLOBAL       3 -4    ; print
        5       [5]     MOVE            4 0
        6       [5]     CALL            3 2 1
        7       [5]     RETURN          0 1
```

``` 
$ cat example-table.lua 
local rcode = {
        one     = 1,
        two     = 2,
        three   = 3
}

print(rcode.one)

$ luac -l example-table.lua 
main <example-table.lua:0,0> (8 instructions, 32 bytes at 0x556389880550)
0+ params, 3 slots, 0 upvalues, 1 local, 7 constants, 0 functions
        1       [1]     NEWTABLE        0 0 3
        2       [2]     SETTABLE        0 -1 -2 ; "one" 1
        3       [3]     SETTABLE        0 -3 -4 ; "two" 2
        4       [4]     SETTABLE        0 -5 -6 ; "three" 3
        5       [7]     GETGLOBAL       1 -7    ; print
        6       [7]     GETTABLE        2 0 -1  ; "one"
        7       [7]     CALL            1 2 1
        8       [7]     RETURN          0 1
```